docker/tests/curl/Dockerfile

```diff
@@ -0,0 +1,80 @@
+# OpenJDK distributed under GPLv2+Oracle Classpath Exception license (http://openjdk.java.net/legal/gplv2+ce.html)
+# Alpine Linux packages distributed under various licenses including GPL-3.0+ (https://pkgs.alpinelinux.org/packages)
+FROM gcr.io/cicd-246518/tests-seed as prepare-curl
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Always tag the version of an image explicitly](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5442148)